tests/autoanalyze.test/runit

```diff
@@ -365,4 +365,40 @@ fi
 #can also test downgrade master until it is a different node
 #stats should be the same as before
 
+echo ""
+echo "Testing Auto Analyze Subtraction Bug"
+get_autoanalyze_stat $master
+stats_should_be $savedCnt $lastRun 0 $lastRun # initial condition: start with savedCnt == 0
+prevDate=$lastRun
```

Comment:
Line 358 (starts with "read cnt") messes Github and VSCode up and make them think that everything past here is a comment. The code still runs though. They don't think it's a comment anymore when you remove the right bracket in "${CDB2_OPTIONS}" on line 358. Idk why though. If you know plz advise thx. Probably Microsoft bug.